docs/content/error/$compile/noident.ngdoc

```diff
@@ -0,0 +1,71 @@
+@ngdoc error
+@name $compile:noident
+@fullName Controller alias is required.
+@description
+
+When using the `bindToController` feature of AngularJS, a directive is required
+to have a Controller identifier. This can be supplied using the "controllerAs"
+property of the directive object, or alternatively by adding " as ALIAS" to the
+controller name (in either the "controller" property, or where the controller
+was registered).
```

Comment:
I am not sure. I don't think you can specify `controllerName as alias` when defining a controller. The only other time would be inside the `ngController` directive, where the controller name is `'@'` and so we get the controller from the `ng-controller` attribute value, which obviously can also have the `controllerName as alias`.

This phrase should either be clarified or removed.